akka-docs/src/main/paradox/typed/actors.md

```diff
@@ -241,8 +241,8 @@ alternatively to `SessionGranted` we may also receive a
 @@@
 
 Now to try things out we must start both a chat room and a gabbler and of
-course we do this inside an Actor system. Since there can be only one guardian
-supervisor we could either start the chat room from the gabbler (which we don’t
+course we do this inside an Actor system. Since there can be only one user guardian
```

Comment:
maybe remove "of course" since it might insult the reader

Reply:
Created a separate issue to fix throughout the docs #27792

Reply:
Created a separate issue to fix throughout the docs #27792